README.md

```diff
@@ -9,3 +9,29 @@ For more information about the control container, including how to use it and ho
 
 You'll need Docker 17.06.2 or later, for multi-stage build support.
 Then run `make`!
+
+## Connecting to AWS Systems Manager (SSM)
+
+Starting from v0.5.0, users have the option to pass in their own activations for SSM. This is for users that want to set up on-premises virtual machines (VMs) in their hybrid environment as managed instances.
```

Comment:
Our convention has been to write markdown files with one sentence per line.  If there are no empty lines, they get formatted as a paragraph.  If you want a new paragraph, leave a blank line.  It just makes these docs a bit easier to read/edit.